packages/notifications/src/error.test.ts

```
@@ -0,0 +1,121 @@
+import { IgnoreErrorDecider, makeNotificationsContract } from './error';
+import { Notifier } from './useNotifications';
+
+const mockNotifications: Notifier = {
+  notify: jest.fn(),
+};
+
+const ignoreError: IgnoreErrorDecider = (title: string, _obj?: unknown): boolean => {
+  return title === 'Ignore Me';
+};
+describe('reportError', () => {
+  it('calls notify and console.error', async () => {
+    // Arrange
+    jest.spyOn(console, 'error').mockImplementation(() => {});
+    const { reportError } = makeNotificationsContract(mockNotifications, ignoreError);
+
+    // Act
+    await reportError('Things went BOOM', new Error('BOOM!'));
+
+    // Assert
+    const expectedError = new Error('BOOM!');
+    expect(mockNotifications.notify).toBeCalledTimes(1);
+    expect(mockNotifications.notify).toBeCalledWith('error', 'Things went BOOM', { detail: expectedError });
+    expect(console.error).toBeCalledTimes(1);
+    expect(console.error).toBeCalledWith('Things went BOOM', expectedError);
+  });
+
+  it('ignores session timeout error', async () => {
```

Where is the session timeout coming from? Is it worth adding a test confirming that the session timeout actually happens?

---

there's a test over in terra-ui app land that tests a part of this more specific to session timeout error check... here it should be renamed to something more generic to ignoring an error.